chart/README.md

```diff
@@ -0,0 +1,501 @@
+# Deploy Sail-Operator by using Helm charts
```

Comment:
JFYI, we are using the convention Sail Operator in our docs.

```suggestion
# Deploy Sail Operator by using Helm charts
```

Reply:
Fixed